apriltag_ros/include/apriltag_ros/continuous_detector.h

```diff
@@ -81,6 +98,24 @@ class ContinuousDetector: public nodelet::Nodelet
 
   ros::ServiceServer refresh_params_service_;
   bool refreshParamsCallback(std_srvs::Empty::Request& req, std_srvs::Empty::Response& res);
+
+  // publishes an empty detection
+  void publishEmptyDetection(const std_msgs::Header& header);
+
+  // helper to convert tf::Vector3 to cv::Point3f
+  cv::Point3f toCvPoint3f(const tf::Vector3& position);
+
+  // checks if a tag is in the FOV of the camera given the tag pose in optical frame
+  bool isTagInFOV(const tf::Transform& tag_pose);
```

Comment:
using the `toCvPoint3f` method above, I was required to remove const from the `isTagInFOV` method. I think that's fine

Reply:
See previous comment.